### ISSUE TITLE
Crash on startup with Accessibility enabled

### ISSUE THREAD

**Issue by mrward** [mono/monodevelop#7138]
Latest master build of VSMac crashes on startup with accessibility enabled. The crash has `[ACAccessibilityElement accessibilityLinkedUIElements:]: unrecognized selector sent to instance 0x7fd1a5b265d0` and the only recent change I can see involving LinkedUIElements is PR https://github.com/mono/monodevelop/pull/7062/

VS Mac: 8.0.0.1454
Git revision: 0c14d046f892155855ab0d04195790342439c776
Build date: 2019-02-12 22:44:27+00
Build branch: master
Xamarin extensions: e7a9ebcc3918c6ae6dc6c74ddac0f0535b9b4c47

Mac OS X 10.13.6
Mono 5.18.0.248 (2018-08/a4956c837e1) (64-bit)

```
2019-02-13 10:18:00.704 VisualStudio[1235:58360] -[ACAccessibilityElement accessibilityLinkedUIElements:]: unrecognized selector sent to instance 0x7fd1a5b265d0
2019-02-13 10:18:00.705 VisualStudio[1235:58360] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[ACAccessibilityElement accessibilityLinkedUIElements:]: unrecognized selector sent to instance 0x7fd1a5b265d0'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff4c03b23b __exceptionPreprocess + 171
	1   libobjc.A.dylib                     0x00007fff732cdc76 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff4c0d3d94 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
	3   CoreFoundation                      0x00007fff4bfb1700 ___forwarding___ + 1456
	4   CoreFoundation                      0x00007fff4bfb10c8 _CF_forwarding_prep_0 + 120
	5   ???                                 0x0000000116aaf795 0x0 + 4675270549
	6   ???                                 0x0000000116aa638b 0x0 + 4675232651
	7   ???                                 0x0000000116aa3973 0x0 + 4675221875
	8   ???                                 0x0000000116aa386b 0x0 + 4675221611
	9   ???                                 0x0000000116aa2d83 0x0 + 4675218819
	10  ???                                 0x0000000106adde03 0x0 + 4407025155
	11  ???                                 0x0000000106ada1fb 0x0 + 4407009787
	12  libmonosgen-2.0.dylib               0x0000000105d9b1e3 mono_jit_runtime_invoke + 1619
	13  libmonosgen-2.0.dylib               0x0000000105f998a4 do_runtime_invoke + 84
	14  libmonosgen-2.0.dylib               0x0000000105f9d37c do_exec_main_checked + 156
	15  libmonosgen-2.0.dylib               0x0000000105e021ad mono_jit_exec + 349
	16  libmonosgen-2.0.dylib               0x0000000105e0538f mono_main + 11615
	17  VisualStudio                        0x00000001033287de main + 3054
	18  libdyld.dylib                       0x00007fff73ee7015 start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException

=================================================================
	Native Crash Reporting
=================================================================
Got a SIGABRT while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries 
used by your application.
=================================================================

=================================================================
	Basic Fault Adddress Reporting
=================================================================
Memory around native instruction pointer (0x7fff74037b66):0x7fff74037b56  ff ff c3 90 90 90 b8 48 01 00 02 49 89 ca 0f 05  .......H...I....
0x7fff74037b66  73 08 48 89 c7 e9 79 6f ff ff c3 90 90 90 b8 4c  s.H...yo.......L
0x7fff74037b76  01 00 02 49 89 ca 0f 05 73 08 48 89 c7 e9 61 6f  ...I....s.H...ao
0x7fff74037b86  ff ff c3 90 90 90 b8 49 01 00 02 49 89 ca 0f 05  .......I...I....

=================================================================
	Native stacktrace:
=================================================================
	0x105e9eb48 - /Library/Frameworks/Mono.framework/Libraries/libmonosgen-2.0.dylib : mono_dump_native_crash_info
	0x105e42425 - /Library/Frameworks/Mono.framework/Libraries/libmonosgen-2.0.dylib : mono_handle_native_crash
	0x7fff741f5f5a - /usr/lib/system/libsystem_platform.dylib : _sigtramp
	0x20008 - Unknown
	0x7fff73f931ae - /usr/lib/system/libsystem_c.dylib : abort
	0x7fff71e8df8f - /usr/lib/libc++abi.dylib : __cxa_bad_cast
	0x7fff71e8e12b - /usr/lib/libc++abi.dylib : _ZL26default_unexpected_handlerv
	0x7fff732cfea3 - /usr/lib/libobjc.A.dylib : _ZL15_objc_terminatev
	0x7fff71ea97c9 - /usr/lib/libc++abi.dylib : _ZSt11__terminatePFvvE
	0x7fff71ea926f - /usr/lib/libc++abi.dylib : _ZN10__cxxabiv1L22exception_cleanup_funcE19_Unwind_Reason_CodeP17_Unwind_Exception
	0x7fff732cdda5 - /usr/lib/libobjc.A.dylib : objc_exception_throw
	0x7fff4c0d3d94 - /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation : -[NSObject(NSObject) doesNotRecognizeSelector:]
	0x7fff4bfb1700 - /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation : ___forwarding___
	0x7fff4bfb10c8 - /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation : _CF_forwarding_prep_0
	0x116aaf795 - Unknown
	0x116aa638b - Unknown
	0x116aa3973 - Unknown
	0x116aa386b - Unknown
	0x116aa2d83 - Unknown
	0x106adde03 - Unknown
	0x106ada1fb - Unknown
	0x105d9b1e3 - /Library/Frameworks/Mono.framework/Libraries/libmonosgen-2.0.dylib : mono_jit_runtime_invoke
	0x105f998a4 - /Library/Frameworks/Mono.framework/Libraries/libmonosgen-2.0.dylib : do_runtime_invoke
	0x105f9d37c - /Library/Frameworks/Mono.framework/Libraries/libmonosgen-2.0.dylib : do_exec_main_checked
	0x105e021ad - /Library/Frameworks/Mono.framework/Libraries/libmonosgen-2.0.dylib : mono_jit_exec
	0x105e0538f - /Library/Frameworks/Mono.framework/Libraries/libmonosgen-2.0.dylib : mono_main
	0x1033287de - vsmac-preview-8.0.0.1454/Visual Studio (Preview).app/Contents/MacOS/VisualStudio : main
	0x7fff73ee7015 - /usr/lib/system/libdyld.dylib : start

=================================================================
	Telemetry Dumper:
=================================================================
Pkilling 0x7000034d5000 from 0x7fffac92d380
Pkilling 0x7000046f0000 from 0x7fffac92d380
Pkilling 0x70000598e000 from 0x7fffac92d380
Pkilling 0x7000044ed000 from 0x7fffac92d380
Pkilling 0x70000578b000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x7000042ea000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x700005588000 from 0x7fffac92d380
Pkilling 0x7000040e7000 from 0x7fffac92d380
Pkilling 0x700005385000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x700003ee4000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x700005182000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x700003ce1000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x700004efc000 from 0x7fffac92d380
Pkilling 0x700003ade000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x700004cf9000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x7000038db000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x700004af6000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x7000036d8000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
Pkilling 0x7000048f3000 from 0x7fffac92d380
* Assertion: should not be reached at mini-posix.c:249
* Assertion: should not be reached at mini-posix.c:249
```

> VS bug [#790820](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/790820)

**Comment by netonjm** [comment]
Same problem here

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-8] Comment by **xamarinc**



Fixed in version 8.0.0.1460 (master)



Pull Request #7139 merged by: iain  
 Author: mono  
 Commit: 9e994bb75c84b836ffc48116f678e7d1707223fc (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-10] Comment by **xamarinc**



Fixed in version 8.0.0.2176 (release-8.0)



Pull Request #7139 merged by: iain  
 Author: mono  
 Commit: 9e994bb75c84b836ffc48116f678e7d1707223fc (xamarin/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-14] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #7139 merged by: iain  
 Author: mono  
 Commit: 9e994bb75c84b836ffc48116f678e7d1707223fc (xamarin/monodevelop)
